api/routes/api_routes.py

```diff
@@ -10,29 +10,25 @@
 
 api = Blueprint("api", __name__)
 
-@api.route('/')
-def hello():
-    return 'Welcome to the RESTful API Server for DreamHome!'
-
 @api.route('/api/v1/education/<id>', methods=['GET'])
 def get_education(id):
     return educationcontroller.get_education(id)
 
 @api.route('/api/v1/education', methods=['GET'])
 def home():
     return educationcontroller.index()
-
-@api.route('/api/v1/education', methods=['POST'])
-def add_education():
-    return educationcontroller.add_education()
-
-@api.route('/api/v1/education/<id>', methods=['PUT'])
-def update_education(id):
-    return educationcontroller.update_education(id)
-
-@api.route('/api/v1/education/<id>', methods=['DELETE'])
-def destroy_education(id):
-    return educationcontroller.destroy_education(id)
+#
+# @api.route('/api/v1/education', methods=['POST'])
```

Comment:
I see we are commenting these routes until Iteration2 - where we have admin authorization on these routes